org.eclipse.lyo.oslc4j.parent/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.lyo.oslc4j.core</groupId>
+	<artifactId>core-parent</artifactId>
+	<version>2.4.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<properties>
+	    <maven.compiler.target>1.8</maven.compiler.target>
+	    <maven.compiler.source>1.8</maven.compiler.source>
+		<tycho.version>1.0.0</tycho.version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<p2.repo>http://download.eclipse.org/releases/oxygen</p2.repo>
+	</properties>
+
+	<repositories>
+		<repository>
+			<id>eclipse-p2</id>
+			<url>${p2.repo}</url>
+			<layout>p2</layout>
+		</repository>
+		<repository>
+			<id>acceleo</id>
+			<url>https://repo.eclipse.org/content/repositories/acceleo-releases/</url>
+		</repository>
+	</repositories>
+
+	<pluginRepositories>
+		<pluginRepository>
+			<id>acceleo</id>
+			<url>https://repo.eclipse.org/content/repositories/acceleo-releases/</url>
+		</pluginRepository>
+	</pluginRepositories>
+	<modules>
+	<!--
+	-->
+		<module>../org.eclipse.lyo.repackaging.jersey</module>
```

Comment:
@jadelkhoury  If we compare it with [pom](https://github.com/eclipse/lyo.designer/blob/5c2b0d3f4fa009c248229010c9d35fc7f988e87f/pom.xml#L19) of lyo designer. Their modules are mention in the root pom and not the parent pom. Moreover, we need to specify a relative path here in the pom. I haven't tried cloning the repository and trying myself yet. However, look at this I guess you are getting errors that this  modules are not found?